app/invocation/invocation_targets.tsx

```diff
@@ -83,7 +83,7 @@ export default class TargetsComponent extends React.Component {
 
         {!!this.props.model.succeeded.length && this.props.mode != "failing" && (
           <TargetsCardComponent
-            buildEvents={this.props.model.succeeded}
+            buildEvents={this.props.model.succeeded.concat(this.props.model.skipped)}
```

Comment:
Won't this show the skipped targets as "Succeeded" rather than "Skipped"?

Reply:
Created a separated skipped card with neutral aesthetics & with purple links @bduffany